assets/data/cards_pt-br.json

```
@@ -482,6 +482,72 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "NaN",
+      "description": "A propriedade global NaN é um valor especial que significa Not-A-Number(Não é um número).",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Null",
+      "description": "Null é um literal que representa um valor nulo ou vazio.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Undefined",
+      "description": "O valor global undefined representa um valor indefinido, ou seja, um valor que não foi definido ou atribuído em uma variável.",
```

Essa explicação utiliza uma analogia simples para tornar o conceito mais acessível, veja se faz sentido:
```suggestion
      "description": "O valor global undefined é usado na programação para mostrar que uma variável ainda não recebeu um valor. É como se você tivesse uma caixa, mas ainda não colocou nada dentro dela.",
```

---

Acham que vale adicionar que nem todas as linguagens usam `undefined`? Algo como:

> Enquanto algumas situações em linguagens de programação podem definir um valor padrão para variáveis criadas porém que não receberam algum valor, outras simplesmente usam o conceito de `undefined` ou `NULL`.

Eu escrevi `algumas situações` porque, por exemplo, se criarmos em c
```c
int x;
```

ele poderia setar `x = 0` internamente, o que seria diferente de

```c
int *x;
```